query-service-impl/src/test/java/org/hypertrace/core/query/service/pinot/MigrationTest.java

```diff
@@ -233,36 +269,166 @@ public void testQueryWithDistinctCountAggregation() {
   }
 
   @Test
-  public void testQueryWithContainsKeyValueOperator() {
+  public void testQueryWithEQFilterForMapAttribute() {
     Builder builder = QueryRequest.newBuilder();
-    Expression spanTag = createSimpleAttributeExpression("Span.tags").build();
+    Expression spanTag = createComplexAttributeExpression("Span.tags", "FLAGS").build();
     builder.addSelection(spanTag);
 
-    Expression tag = createStringArrayLiteralValueExpression(List.of("FLAGS", "0"));
-    Filter likeFilter =
+    Filter equalFilter =
         Filter.newBuilder()
-            .setOperator(Operator.CONTAINS_KEYVALUE)
+            .setOperator(Operator.EQ)
             .setLhs(spanTag)
-            .setRhs(tag)
+            .setRhs(createStringLiteralValueExpression("0"))
+            .build();
+    builder.setFilter(
+        createCompositeFilter(
```

Comment:
Do we need to explicitly set this? In other words, is this test only the converter, or does it include the request transformer which should do this for us? If it's only the converter, let's make sure we include some tests that take the whole pipeline together - probably integration tests.